Files/Views/ColumnShellPage.xaml

```diff
@@ -30,6 +30,11 @@
             Invoked="KeyboardAccelerator_Invoked"
             IsEnabled="{x:Bind IsCurrentInstance, Mode=OneWay}"
             Modifiers="Control" />
+        <KeyboardAccelerator
```

Comment:
With the code behind detecting key presses, is this actually doing anything?

Reply:
Yes this is necessary otherwise the function won't be invoked.